addons/dragon/anim/model.cfg

```diff
@@ -0,0 +1,124 @@
+class cfgSkeletons
```

Comment:
Opening brackets on the same line

Reply:
copy of https://github.com/acemod/ACE3/blob/master/addons/gestures/anim/model.cfg
iirc it's from the a3 samples
I don't think it's actually used, pboProject just thinks it's needed